<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewSpriteSheet</class>
 <widget class="QDialog" name="NewSpriteSheet">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Sprite Sheet</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QComboBox" name="cbSheetSize">
       <item>
        <property name="text">
         <string>8x8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16x16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32x32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>64x64</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>128x128</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Tile Size:</string>
       </property>
       <property name="buddy">
        <cstring>cbTileSize</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="cbTileSize">
       <item>
        <property name="text">
         <string>8x8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16x16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32x32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>64x64</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>128x128</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>&amp;Sheet Size(Tiles):</string>
       </property>
       <property name="buddy">
        <cstring>cbSheetSize</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="leName"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Name:</string>
       </property>
       <property name="buddy">
        <cstring>leName</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leName</tabstop>
  <tabstop>cbTileSize</tabstop>
  <tabstop>cbSheetSize</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewSpriteSheet</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewSpriteSheet</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
